src/lib/components/forms/checkbox/example/terra-checkbox.component.example.ts

```diff
@@ -25,7 +25,7 @@ export class TerraCheckboxComponentExample implements OnInit {
         this.viewChildIndeterminateCb.isIndeterminate = !this.viewChildIndeterminateCb.isIndeterminate;
     }
 
-    public _setCheckboxValue(): void {
-        this.viewChildTestCb.value = !this.viewChildTestCb.value;
+    public _setCheckboxValue(value: boolean): void {
+        this.viewChildTestCb.writeValue(!value);
```

Comment:
ich würde eher eine boolean Variable in der TS-Datei erstellen und die halt immer ändern... und im HTML bei der Checkbox halt ngModel verwenden

Reply:
und das ViewChild entfernen

Reply:
gute Idee. das machte es einfacher.